### ISSUE TITLE
2021-11-6

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#647]
281Add cross entropy to database; done; (2021-10-25)
282Prof He ask me to do a presentation, what should I do? As a weekly report, introduce recently report is better; introduce simclr and moco? (2021-10-25)
283Download paper simclr; done; add simclr to database; done; (2021-10-25)
284AG; (2021-10-26)
285What is minibatch? Batch means that you use all your data to compute the gradient during one iteration. Mini-batch means you only take a subset of all your data during one iteration. (2021-10-26)
286What is memory bank? 
287What is BN? (Batch Normalization）; Batch Normalization是2015年一篇论文中提出的数据归一化方法，往往用在深度神经网络中激活层之前。其作用可以加快模型训练时的收敛速度，使得模型训练过程更加稳定，避免梯度爆炸或者梯度消失。并且起到一定的正则化作用，几乎代替了Dropout。
288Download ResNet paper; done; add ResNet to database; done; (2021-10-26)
289Download BN; done; add BN to database; done; (2021-10-26)
